google_symptoms/delphi_google_symptoms/run.py

```diff
@@ -84,14 +83,10 @@ def run_module(params):
                        export_start_date,
                        export_end_date,
                        num_export_days)
-    gmpr = geomap.GeoMapper()
 
     for geo_res in GEO_RESOLUTIONS:
         if geo_res == "state":
             df_pull = dfs["state"]
-        elif geo_res in ["hhs", "nation"]:
-            df_pull = gmpr.replace_geocode(dfs["county"], "fips", geo_res, from_col="geo_id")
-            df_pull.rename(columns={geo_res: "geo_id"}, inplace=True)
         else:
             df_pull = geo_map(dfs["county"], geo_res)
```

Comment:
No. We already know that at least for the current Google Symptoms (Anosmia and Ageusia only), we have only around ~100 counties available. Since we do a population weighted average, it should be more accurate to do the aggregation from county level. For example, we only have 2 counties available in state A, if we start from state level, we will add a weighted based on the state population to the already normalized and scaled count from Google.

Reply:
Or do you think the other way is more reasonable?

Reply:
_we_ only have two counties available in state A, but we have no idea how many counties go into _Google's_ value for state A, right? if they're able to include more counties in their published state figures, shouldn't we make use of them?

Reply:
Yeah that makes sense, I may make too far assumption  about this. Since a lot of things are unknown to us, I think you and Ryan is correct. I will change those mappings from state level.

Reply:
I can see there are pros/cons either way, but I do think it's worth aggregating state signals to build national and HHS signals, for ultimately the following reason: state numbers are based on normalizing state counts, and there's a lot less censorship here than at the county level (where lots of counts are censored), so you'll be able to leverage a lot more data this way.